.github/workflows/branches-and-prs.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         os: ['ubuntu-latest']
         variant: ['2.5', '3.0']
-        java: ['8', '11', '15']
+        java: ['8', '11', '16']
```

Comment:
Groovy 2.5 doesn't seem to support Java 16. You should probably exclude those builds.

Reply:
I would not exclude, but configure to compile to a lower target. Running the build on a recent JDK is still important, IMO.

Reply:
You might be right. Groovy 2.5 might be able to be executed with Java 16, but with lower target version (but I haven't tested it).

Reply:
I upgraded my Spock playground project, and indeed it is working, despite a few compiler quirks in Groovy-Eclipse which forced me to refactor two classes. Also IntelliJ IDEA is bitching around now. But I got it compiling in Maven. Please also refer to
  * https://github.com/groovy/groovy-eclipse/issues/1292
  * https://youtrack.jetbrains.com/issue/IDEA-275889

But in general, it should work.